### DISCUSSION TITLE
error when trying to close browser

### DISCUSSION THREAD

**Discussion by rafafranco2203** [ultrafunkamsterdam/undetected-chromedriver#1808] [General]
It's giving me an error every time I try to close the browser
from auto_download_undetected_chromedriver import download_undetected_chromedriver
import undetected_chromedriver as uc
from time import sleep

folder_path = "c:\\download2thisfolder20"
chromedriver_path = download_undetected_chromedriver(folder_path, undetected=True, arm=False, force_update=False)

browser = uc.Chrome(driver_executable_path=chromedriver_path, headless=False, use_subprocess=True)
browser.set_window_size(710, 700)
browser.get('https://www.tiktok.com/login')
browser.set_page_load_timeout(60)
sleep(10)
browser.quit()

PS C:\Users\rafa\Desktop\projetos\PANDA-TikTok> & C:/Users/rafa/AppData/Local/Programs/Python/Python312/python.exe c:/Users/rafa/Desktop/projetos/PANDA-TikTok/teste.py
To start chromedriver: uc.Chrome(driver_executable_path=c:\download2thisfolder20\chromedriver.exe)
Exception ignored in: <function Chrome.__del__ at 0x00000223A2AE9760>
Traceback (most recent call last):
  File "C:\Users\rafa\AppData\Local\Programs\Python\Python312\Lib\site-packages\undetected_chromedriver\__init__.py", line 843, in __del__
  File "C:\Users\rafa\AppData\Local\Programs\Python\Python312\Lib\site-packages\undetected_chromedriver\__init__.py", line 798, in quit   
OSError: [WinError 6] Identificador inválido
PS C:\Users\rafa\Desktop\projetos\PANDA-TikTok>

**Comment by KebabLord** [comment]
This is a known bug with UC that some users are experiencing, #1339

This error doesn't affect the program so you can safely surpress it by:
```
try:
    browser.quit()
except OSError:
    pass
```
